torch/serialization.py

```diff
@@ -748,11 +748,11 @@ def persistent_load(saved_id):
         # only if offset is zero we can attempt the legacy tar file loader
         try:
             return legacy_load(f)
-        except tarfile.TarError:
+        except tarfile.TarError as e:
```

Comment:
Nit (`e` is not used here)
```suggestion
        except tarfile.TarError:
```